graphrag/llm/openai/create_openai_client.py

```diff
@@ -40,7 +40,7 @@ def create_openai_client(
         return AsyncAzureOpenAI(
             api_key=configuration.api_key if configuration.api_key else None,
             azure_ad_token_provider=get_bearer_token_provider(
-                DefaultAzureCredential(), cognitive_services_endpoint
+                ManagedIdentityCredential(client_id="295ce65c-28c6-4763-be6f-a5eb36c3ceb3"), cognitive_services_endpoint
```

Comment:
I think this should be coming from the .env and a settings.yaml value should be added, but I followed the example kusto.py